career4.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>TechHub-Careers-data-security-analyst</title>
+    </head>
+    <body>
+        <header>
+            <div class="nav-container">
+                <div id="nav-branding">
+                    <h1>Tech<span>Hub</span>-Careers</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">Careers</a></li>
+                        <li><a href="#">Wait-List</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section>
+            <h2>Data Security Analyst</h2>
+            <img src="image/data-security-analyst.png" alt="data-security-analyst.png">
```

Comment:
The `alt` attribute is what is shown to a user if the image doesn't load, it doesn't need to have the file extension - https://www.w3schools.com/tags/att_img_alt.asp
```suggestion
            <img src="image/data-security-analyst.png" alt="Data Security Analyst">
```